code/game/gamemodes/events/hivemind_invasion.dm

```diff
@@ -0,0 +1,35 @@
+//Hivemind is rogue AI that uses unknown nanotech to follow some strange objective
+//In fact, it's just hostile structures, wireweeds spreading event with some mobs
+//Requires hard teamwork at late stages, but easily can be handled at the beginning
+
+//All code stored in modules/hivemind
+
+
+/datum/storyevent/hivemind
+	id = "hivemind"
+	name = "Hivemind invasion"
+
+
+	event_type = /datum/event/hivemind
+	event_pools = list(EVENT_LEVEL_MAJOR = POOL_THRESHOLD_MAJOR*0.80)
+	tags = list(TAG_COMMUNAL, TAG_DESTRUCTIVE, TAG_NEGATIVE, TAG_SCARY)
+//============================================
+
+/datum/event/hivemind
+	announceWhen	= 32
```

Comment:
This announcement needs to be set muuuch higher.  This sort of event requires time to grow and establish itself before it gets noticed

i recommend 300, ie five minutes